openslides_backend/action/actions/projector/add_to_preview.py

```diff
@@ -0,0 +1,55 @@
+from ....models.models import Projection, Projector
+from ....shared.filters import FilterOperator
+from ....shared.patterns import Collection, string_to_fqid
+from ....shared.schema import required_id_schema
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData
+from ..projection.create import ProjectionCreate
+
+
+@register_action("projector.add_to_preview")
+class ProjectorProject(UpdateAction):
+    """
+    Action to projector project.
+    """
+
+    model = Projector()
+    schema = DefaultSchema(Projection()).get_default_schema(
+        required_properties=["content_object_id"],
+        optional_properties=["options", "stable", "type"],
+        additional_required_fields={
+            "ids": {"type": "array", "items": required_id_schema}
+        },
+        title="Projector project schema",
+    )
+
+    def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        for instance in action_data:
+            # check content object and get meeting id.
+            content_object = self.datastore.get(
+                string_to_fqid(instance["content_object_id"]), ["meeting_id"]
+            )
+            meeting_id = content_object["meeting_id"]
+
+            # add the preview projections
+            for projector_id in instance["ids"]:
```

Comment:
is it checked, that the projector ids belong to the meeting? Maybe add a test for this.

Reply:
Yes, it is indirectly checked. Added a test for this. You cannot add a projection where meeting_id differ with content_object meeting_id.